include/SDL3/SDL_oldnames.h

```diff
@@ -474,6 +474,8 @@
 #define SDL_TLSCreate SDL_CreateTLS
 #define SDL_TLSGet SDL_GetTLS
 #define SDL_TLSSet SDL_SetTLS
+#define SDL_ThreadID SDL_GetCurrentThreadID
+#define SDL_threadID SDL_ThreadID
```

Comment:
Unfortunately this will end up expanding `SDL_threadID` to `SDL_GetCurrentThreadID`.
Not sure what the best solution is though, apart from adding a special case in rename_api to output a `typedef unsigned long SDL_threadID` instead?